.storybook/Image.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withInfo } from '@storybook/addon-info'
+import { Image, Box } from '../src'
+
+const description = 'A low-level layout component that renders an image'
+
+storiesOf('Image', module)
+  .add('Image component', withInfo({
+    text: description,
+    inline: true
+  })(() => (
+    <Image src='https://www.priceline.com/home/public/assets/images/photos/photo-aruba.jpg' />
+  )))
+
+  .add('Fixed width', () => (
+    <Image
+      src='https://www.priceline.com/home/public/assets/images/photos/photo-aruba.jpg'
+      width='300px'
```

Comment:
The `width` and `height` attributes are unitless pixel values – different from CSS. The `width` and `height` attributes should **always** be set on images to help with page layout and prevent reflows when images load. CSS values can override these HTML attributes, if needed, for responsive sizing, but generally that can be handled with wrapping layout components like `<Box />`